backend/src/middleware/hashtags/extractHashtags.spec.js

```diff
@@ -8,9 +8,24 @@ describe('extractHashtags', () => {
   })
 
   describe('searches through links', () => {
-    it('finds links with and without ".hashtag" class and extracts Hashtag names', () => {
-      const content =
-        '<p><a class="hashtag" href="/search/hashtag/Elections">#Elections</a><a href="/search/hashtag/Democracy">#Democracy</a></p>'
+    it('not `class="hashtag"` but `data-hashtag-id="something"` makes a link a hashtag link', () => {
```

Comment:
This sounds nice, but is not true in real.
Because the `data-hashtag-id` is deleted by saving the content in the database.
On reedit it is no link anymore in this sense.